Gemfile

```diff
@@ -5,10 +5,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby(File.read(File.join(File.dirname(__FILE__), '.ruby-version')).strip)
 
 gem 'bootsnap', '>= 1.4.4', require: false
+gem 'draper', '~> 4.0.0'
 gem 'factory_bot_rails', '~>6.2.0'
 gem 'ffaker', '~> 2.21.0'
 gem 'haml-rails', '~> 2.0'
+gem 'pagy', '~> 5.10'
 gem 'pg', '~> 1.1'
+gem 'pry'
```

Comment:
pry-rails and move it to dev and test env group

Reply:
fixed